src/Docs/Resources/current/4-how-to/270-custom-module.md

```diff
@@ -198,6 +221,9 @@ Shopware.Module.register('custom-module', {
 As mentioned above, Shopware 6 is looking for a `main.js` file in your plugin.
 Its contents get minified into a new file named after your plugin and will be moved to the `public` directory
 of the Shopware 6 root directory.
+
+Prior to shopware 6.1.3 you need the following steps to make sure everything is loaded properly.
```

Comment:
if this is not needed anymore. I think we could completely remove it

Reply:
But maybe some guys are still stuck and can relate how to migrate this. I mean sure we can drop it but I'd like to keep it at least a while